pkg/v5/resource/chartconfig/desired.go

```diff
@@ -72,9 +73,13 @@ func newIngressControllerChartConfig(clusterConfig cluster.Config, projectName s
 		Spec: v1alpha1.ChartConfigSpec{
 			Chart: v1alpha1.ChartConfigSpecChart{
 				Name:      chartName,
-				Channel:   channelName,
 				Namespace: apismetav1.NamespaceSystem,
-				Release:   releaseName,
+				Channel:   channelName,
+				ConfigMap: v1alpha1.ChartConfigSpecConfigMap{
+					Name:      configMapName,
+					Namespace: apismetav1.NamespaceSystem,
+				},
```

Comment:
Configures the config map. Setting the resource version to trigger updates will come in the next PR.